public/components/event_analytics/explorer/no_results.tsx

```diff
@@ -127,7 +132,9 @@ export const NoResults = ({ tabId }: any) => {
                     <p>Show a list of tables within a database</p>
                     <EuiSpacer size="s" />
                     <CreatedCodeBlock
-                      code={`SHOW TABLE EXTENDED IN ${datasourceName}.<database> LIKE '*'`}
+                      code={`SHOW ${
+                        dataSourceConnectionType === 'SECURITYLAKE' ? 'TABLES' : 'TABLE EXTENDED'
```

Comment:
if this is just `show tables` query, do we want to filter them based on hive or non-hive provider?

Reply:
Security lake only supports Iceberg tables so we don't need any filtering